experimental/apps-mcp/README.md

```diff
@@ -0,0 +1,368 @@
+# Databricks MCP Server
```

Comment:
Non-blocking: this doc should be reviewed, it's actually an app-specific MCP version